clients/drcachesim/tracer/output.cpp

```diff
@@ -1169,6 +1205,16 @@ init_thread_io(void *drcontext)
         BUF_PTR(data->seg_base) +=
             append_unit_header(drcontext, BUF_PTR(data->seg_base),
                                dr_get_thread_id(drcontext), get_local_window(data));
+        if (op_L0_filter_until_instrs.get_value()) {
+            // If we have switched to instruction trace already, then add a
+            // FILTER_ENDPOINT marker.
```

Comment:
Hmm, so new threads created a long time after the transition still have this marker?  I guess that's fine; should it be explicitly documented?

Reply:
Mentioned this in the comment for OFFLINE_FILE_TYPE_IFILTERED